.github/workflows/fuzz.yml

```diff
@@ -13,13 +13,22 @@ jobs:
       run: |
         set -e
         rustup set profile minimal
-        rustup toolchain install nightly 
+        rustup toolchain install nightly
         rustup default nightly
     - uses: taiki-e/install-action@v2
-      with: 
+      with:
         tool: cargo-bolero
     - run: |
         set -e
-        (cd profiling && cargo bolero test collections::string_table::tests::fuzz_string_table -T 1min)
-        (cd profiling && cargo bolero test collections::string_table::tests::fuzz_arena_allocator -T 1min)
+        DIRS="profiling"
+        for dir in $DIRS;
+        do
+          cd $dir && cargo bolero list | \
+          grep -oP '"(package|test)"\s*:\s*"\K[^"]+' | \
```

Comment:
A comment showing the expected output, and how this matches, will help the future maintainer